tests/unit/multi_table/test_base.py

```diff
@@ -342,3 +342,93 @@ def test_validate_missing_foreign_keys(self):
         )
         with pytest.raises(InvalidDataError, match=error_msg):
             instance.validate(data)
+
+    def test_auto_assign_transformers(self):
+        """Test that each table of the data calls its single table auto assign method."""
+        # Setup
+        metadata = get_multi_table_metadata()
+        instance = BaseMultiTableSynthesizer(metadata)
+        table1 = pd.DataFrame({'col1': [1, 2]})
+        table2 = pd.DataFrame({'col2': [1, 2]})
+        data = {
+            'nesreca': table1,
+            'oseba': table2
+        }
+        instance._table_synthesizers['nesreca'] = Mock()
+        instance._table_synthesizers['oseba'] = Mock()
+
+        # Run
+        instance.auto_assign_transformers(data)
+
+        # Assert
+        instance._table_synthesizers['nesreca'].auto_assign_transformers.assert_called_once_with(
+            table1)
+        instance._table_synthesizers['oseba'].auto_assign_transformers.assert_called_once_with(
+            table2)
+
+    def test_auto_assign_transformers_error(self):
```

Comment:
minor: but for all these can we make the suffix `missing_table` instead of `error`